clients/gtest/matmul_gtest.yaml

```diff
@@ -622,9 +621,8 @@ Tests:
   scaleB: [0, 1]
   scaleC: [0, 1]
   scaleD: [0, 1]
-  # TODO- support bias types other than Tscale or To
-  # bias_vector: [0, 1]
-  # bias_type: f16_r
+  bias_vector: [0, 1]
+  bias_type: f16_r
   unit_check: 1
```

Comment:
The test cases can test for bias type != To or Talpha. And this is also the default bias type for F8F8S gemm